src/mjolnir/convert_transit.cc

```diff
@@ -1055,7 +1144,7 @@ void build_tiles(const boost::property_tree::ptree& pt,
     // a writeable instance (deserialize it so we can add to it)
     lock.lock();
 
-    GraphId transit_tile_id = GraphId(tile_id.tileid(), tile_id.level() + 1, tile_id.id());
+    GraphId transit_tile_id = GraphId(tile_id.tileid(), tile_id.level(), tile_id.id());
```

Comment:
ok so outside of this function we changed it to loop over transit tile ids now (ie level 3) so we no longer need this